### PR TITLE
Updating the dockerfile with the new LD_LIBRARY_PATH.

### PR DESCRIPTION
**Pull request by av8ramit**

### CODE REVIEW

**Comment by gunan** [comment]
Could you test to see if this fixes our problem?

**Comment by av8ramit** [comment]
http://ci.tensorflow.org/view/Experimental/job/experimental-docker-gpu/

**Comment by av8ramit** [comment]
http://ci.tensorflow.org/view/Experimental/job/experimental-docker-gpu/TF_DOCKER_BUILD_IS_DEVEL=YES,TF_DOCKER_BUILD_PYTHON_VERSION=PYTHON3,label=gpu-linux/1/consoleFull

Seems to have an issue.

File: tensorflow/tools/docker/Dockerfile.devel-gpu [hunk 1]
```diff
@@ -78,12 +78,12 @@ WORKDIR /tensorflow
...
 # Configure the build for our CUDA configuration.
 ENV CI_BUILD_PYTHON python
-ENV LD_LIBRARY_PATH /usr/local/cuda/extras/CUPTI/lib64:$LD_LIBRARY_PATH
+ENV LD_LIBRARY_PATH /usr/local/cuda/lib64:$LD_LIBRARY_PATH
 ENV TF_NEED_CUDA 1
 ENV TF_CUDA_COMPUTE_CAPABILITIES=3.0,3.5,5.2,6.0,6.1
 
 RUN tensorflow/tools/ci_build/builds/configured GPU \
-    bazel build -c opt --config=cuda --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" \
+    bazel build -c opt --config=cuda action_env=LD_LIBRARY_PATH=${LD_LIBRARY_PATH} --cxxopt="-D_GLIBCXX_USE_CXX11_ABI=0" \
```
Comment (allenlavoie) [inline tensorflow/tools/docker/Dockerfile.devel-gpu]: "--action_env" rather than "action_env"

Comment (av8ramit) [reply tensorflow/tools/docker/Dockerfile.devel-gpu]: Whoops, typo. I'll fix it and test.

Comment (av8ramit) [reply tensorflow/tools/docker/Dockerfile.devel-gpu]: http://ci.tensorflow.org/view/Experimental/job/experimental-docker-gpu/3/

**Comment by gunan** [comment]
Looks like there are still failures?

**Comment by av8ramit** [comment]
Yes, --action_env didn't work; @allenlavoie is investigating.

**Comment by allenlavoie** [comment]
I'm (still) looking at it. I can comment out the Bazel command, build the image, then enter the Docker container manually (nvidia-docker -it) and build the pip package fine (same flags). I haven't gotten it to run with the Docker build script, mostly because I change something and spend a few hours compiling CUDA kernels.

**Comment by allenlavoie** [comment]
If I add "RUN ls /usr/local/nvidia/lib64/libcuda.so.1", that also fails (and does it much quicker than compiling TensorFlow). But the final image (nvidia-docker -it) seems to have it?

**Comment by gunan** [comment]
Maybe somehow we are not using nvidia-docker when building the image?

**Comment by allenlavoie** [comment]
Ah, so adding "RUN ln -s /usr/local/cuda/lib64/stubs/libcuda.so /usr/local/cuda/lib64/stubs/libcuda.so.1", then adding /usr/local/cuda/lib64/stubs/ to LD_LIBRARY_PATH should work. Looks like there's some file copying that doesn't happen until after. Will try a compile that way.

**Comment by allenlavoie** [comment]
Woo, that worked for me locally. I did not need an action_env (I guess LD_LIBRARY_PATH does get picked up, it was just useless in this case).

**Comment by allenlavoie** [comment]
I've opened https://github.com/tensorflow/tensorflow/pull/13399 (but feel free to ignore it and modify this one if you'd prefer).

**Comment by gunan** [comment]
Fixed via #13399

**Pull request closed without merging**